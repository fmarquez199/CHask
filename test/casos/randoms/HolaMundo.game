world %HolaMundo%:
  drop ~Hello World!~
.~
