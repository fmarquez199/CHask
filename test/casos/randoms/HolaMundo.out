Token world en la fila: 1, columna: 1
Token nombre programa %HolaMundo% en la fila: 1, columna: 7
Token : en la fila: 1, columna: 18
Token drop en la fila: 2, columna: 3
Token string ~Hello World!~ en la fila: 2, columna: 8
Token .~ en la fila: 3, columna: 1
